miscellaneous/eggnog_performance.md

```diff
@@ -15,4 +15,34 @@ The results are summarized in the table below:
 | 5               | 1                             | N.A. (43 finished)        | 21 min                       |
 | 10              | 1                             | N.A. (39 finished)        | 42 min                       |
 
-Each node has 3 hrs of wall time.
\ No newline at end of file
+(Each node has 3 hrs of wall time.)
+
+### Test with scratch drive (Recommended by NERSC)
+
+Copied the source files to `$SCRATCH` and ran the same tests. The results are summarized in the table below:
+
+| Number of nodes | Parallelization within a node | Time to finish 50 genomes                         | Average node time per genome |
+|-----------------|-------------------------------|---------------------------------------------------|------------------------------|
+| 1               | 1                             | 54.83 mins                                        | 1.1 min                      |
+| 1               | 2                             | 39.77 mins (both batches)                         | 0.79 min                     |
+| 1               | 5                             | 27.92 mins (first batch), 37.11 mins (last batch) | 0.74 min                     |
+| 1               | 10                            | 22.62 mins (first batch), 34.7 mins (last batch)  | 0.69 min                     |
+| 1               | 15                            | 18.8 mins (first batch), 30.79 mins (last batch)  | 0.62 min                     |
+| 2               | 1                             | 25.91 mins (first node), 26.85 mins (last node)   | 1.06 min                     |
+| 5               | 1                             | 10.03 mins (first node), 13.24 mins (last node)   | 1.21 min                     |
+
+
+Note: Creating a symbolic link to the source files/tool libraries in the scratch directory does not seem to work,
+as it defeats the purpose of using the scratch area for fast file access.
+Instead, you need to use the `cp` command to copy the files to the scratch directory.
```

Comment:
Does cp take a significant amount of time such that it might offset the time savings from using scratch?

It doesn't count against our allocation though... and if we're moving data from Minio we can probably move it straight to scratch.

Reply:
cp is pretty fast for the 50 genomes we tested + it's not using our allocation time.

Reply:
:+1: